UI/forms/OBSBasicSettings.ui

```diff
@@ -1849,6 +1849,91 @@
                 </layout>
                </widget>
               </item>
+              <item>
+               <widget class="QGroupBox" name="simulcastGroupBox">
+                <property name="title">
+                 <string>Basic.Settings.Stream.SimulcastLabel</string>
+                </property>
+                <layout class="QVBoxLayout" name="verticalLayout_35">
+                 <property name="leftMargin">
+                  <number>9</number>
+                 </property>
+                 <property name="topMargin">
+                  <number>2</number>
+                 </property>
+                 <property name="rightMargin">
+                  <number>9</number>
+                 </property>
+                 <property name="bottomMargin">
+                  <number>9</number>
+                 </property>
+                 <item>
+                  <layout class="QHBoxLayout" name="horizontalLayout_33">
+                   <item>
+                    <spacer name="horizontalSpacer_33">
+                     <property name="orientation">
+                      <enum>Qt::Horizontal</enum>
+                     </property>
+                     <property name="sizeType">
+                      <enum>QSizePolicy::Fixed</enum>
+                     </property>
+                     <property name="sizeHint" stdset="0">
+                      <size>
+                       <width>170</width>
+                       <height>10</height>
+                      </size>
+                     </property>
+                    </spacer>
+                   </item>
+                   <item>
+                    <widget class="QLabel" name="simulcastInfo">
```

Comment:
nit: it might be useful to put all of simulcast under a toggle. That way users don't have to think about an advanced setting even if the default is only the 1 layer.

Reply:
I don't see this as an advanced setting. If you are streaming you are probably going to set this to more then 1 (because your streaming provider suggests it)

Reply:
So the default flow is going to have an additional click.